Dockerfile

```diff
@@ -1,17 +0,0 @@
-FROM golang
```

Comment:
Maybe we need a tracking issue to get this back in at some point.

Reply:
Added #106

Reply:
thanks!